target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/dsa.c

```diff
@@ -2495,6 +2495,118 @@ static int rtldsa_phy_write(struct dsa_switch *ds, int addr, int regnum, u16 val
 	return mdiobus_write_nested(priv->parent_bus, addr, regnum, val);
 }
 
+static const struct flow_action_entry *rtlsa_rate_policy_extract(struct flow_cls_offload *cls)
```

Comment:
Should this be rtl**d**sa_rate_policy_extract? (prefix of `rtldsa`, instead of `rtlsa` as currently)

Reply:
thanks, fixed